docker-compose.yaml

```diff
@@ -15,6 +15,7 @@ services:
       # forward out edge port. set boto3's `endpoint_url`
       # to be http://localhost:4566 to use localstack for development
       - "4566:4566"
+      - "20005:4566"
```

Comment:
Please correct me, the only 2 changes are this line

Reply:
yes, that is correct

Reply:
yes